Source/WebCore/testing/Internals.cpp

```diff
@@ -571,6 +570,7 @@ void Internals::resetToConsistentState(Page& page)
         return;
 
     localMainFrame->setTextZoomFactor(1.0f);
+    page.chrome().client().setPageScaleFactor(1, IntPoint(0, 0), [] { });
```

Comment:
I think this should be removed and completely replaced by UI process initiated resetting.